examples/grpc/setup.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracer } = require('@opentelemetry/node-sdk');
+const { SimpleSpanProcessor } = require('@opentelemetry/basic-tracer');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const EXPORTER = process.env.EXPORTER || '';
+
+function setupTracerAndExporters(service) {
+  let exporter;
+  const options = {
+    serviceName: service
+  };
+  const tracer = new NodeTracer({
+    plugins: {
```

Comment:
Do we need to pass these settings explicitly, AFAIK We load all default plugins in `NodeTracer` or this is only to show and load gRPC plugin settings?

Reply:
I wanted to hide any magic going on behind the scenes. Maybe I could add a comment that all of this additional configuration is optional.

Reply:
related https://github.com/open-telemetry/opentelemetry-js/issues/330